packages/nextjs/components/Mapbox.js

```diff
@@ -1,42 +1,82 @@
 // eslint-disable-line import/no-webpack-loader-syntax
+import React, { useEffect, useRef, useState } from "react";
 import mapboxgl from "!mapbox-gl";
 // import { blo } from "blo";
 import "mapbox-gl/dist/mapbox-gl.css";
-import React, { useEffect, useRef, useState } from "react";
 
 // import { useAccount } from "wagmi";
 
 mapboxgl.accessToken = "pk.eyJ1Ijoicm9uY2h1Y2siLCJhIjoiY2x2Y2o5Z2drMGY3cjJrcGI4b2xsNzdtaCJ9.gi5RJ8qRhTSwfYuhVwhmvQ";
 
 export default function Mapbox({
-  setIsControlsActive = (bool) => bool,
+  setIsControlsActive = bool => bool,
   height = "70vh",
-  setLatLng = (arr) => arr,
+  setLatLng = arr => arr,
   isCheckInActive = false,
-  attestationsData = {},
+  // attestationsData = {},
   latLngAttestation = [],
   setIsLoading = bool => bool,
 }) {
   const mapContainer = useRef(null);
   const map = useRef(null);
   const markersRef = useRef([]);
-  const [lng, setLng] = useState(0.06);
-  const [lat, setLat] = useState(51.03);
+  const [lng, setLng] = useState(0);
+  const [lat, setLat] = useState(0);
   const [zoom, setZoom] = useState(6);
 
-  // const [fakeUsers, setFakeUsers] = useState([]);
-  // const { address } = useAccount();
+  useEffect(() => {
```

Comment:
Best to get these methods out of the `useEffect` let's check this befor the release, will at least add a comment about it on my branch👌